massa-versioning/src/versioning.rs

```diff
@@ -908,11 +908,32 @@ impl MipStoreRaw {
                 {
                     let entry_value = e.get_mut();
                     *entry_value = entry_value.saturating_sub(1);
+                    network_version_count = *entry_value;
                     if *entry_value == 0 {
                         self.stats.network_version_counters.remove(&removed_version);
                     }
                 }
             }
+
+            if announced_network_version != 0 {
+                let vote_ratio = Ratio::new(
+                    network_version_count,
+                    self.stats.config.block_count_considered as u64,
+                );
+
+                if vote_ratio > MIP_STATS_RATIO_WARN_ANNOUNCED_VERSION {
```

Comment:
`MIP_STATS_RATIO_WARN_ANNOUNCED_VERSION` should be a config file parameter and not a constant

Reply:
Done